src/components/editor/Bindings/Editor.tsx

```diff
@@ -125,12 +125,17 @@ function BindingsEditor({ loading, skeleton, readOnly = false }: Props) {
                                 >
                                     <Typography variant="h6" sx={{ mr: 1 }}>
                                         <FormattedMessage id="workflows.collectionSelector.header.collectionSchema" />
+                                        <ExternalLink link="https://docs.estuary.dev/concepts/collections/#specification">
+                                            <FormattedMessage id="terms.documentation" />
+                                        </ExternalLink>
```

Comment:
Now that there are errors like "fix the schema" I think adding a link to the docs is nice